js/private/js_image_layer.bzl

```diff
@@ -18,9 +18,16 @@ load("@bazel_skylib//lib:paths.bzl", "paths")
 
 _DOC = """Create container image layers from js_binary targets.
 
-js_image_layer supports transitioning to specific platform for cross-compiling.
+By design, js_image_layer doesn't have any preference over which rule assembles the container image. 
+This means the downstream rule (`oci_image`, or `container_image` in this case) must set a proper `workdir` and `cmd` to for the container work.
+A proper `cmd` usually looks like /`[ root of js_image_layer ]`/`[ relative path to BUILD file from WORKSPACE or package_name() ]/[ name of js_binary ]`, 
+unless you have a launcher script that invokes the entry_point of the `js_binary` in a different path.
+On the other hand, `workdir` has to be set to `runfiles tree root` which would be exactly `cmd` **but with `.runfiles` suffix**. If `workdir` is not set correctly, some
+attributes such as `chdir` might not work properly.
```

Comment:
The workspace name needs to be included in this path (either what's in WORKSPACE[.bazel] or `__main__` if unset).

Reply:
Note: getting access to the workspace name in a macro is impossible.